android/app/src/main/java/org/fossasia/openevent/core/main/MainActivity.java

```diff
@@ -326,7 +326,7 @@ private void setUpUserProfileMenu() {
     private void setNavHeader(Event event) {
         String logo = event.getLogoUrl();
         if (!Utils.isEmpty(logo)) {
-            StrategyRegistry.getInstance().getHttpStrategy().getPicassoWithCache().load(logo).into(headerView);
+            StrategyRegistry.getInstance().getHttpStrategy().getPicassoWithCache().load(logo).error(R.mipmap.ic_launcher).into(headerView);
```

Comment:
Please move each function call in its own line and do the same below as well

Reply:
@iamareebjamal I have moved each function call to different line.Please Review.